pkg/utils/docker.go

```diff
@@ -39,7 +39,7 @@ var data = `
 version: 2
 services:
  codewind-pfe:
-  image: ${REPOSITORY}codewind-pfe${PLATFORM}:${TAG}
+  image: eclipse/codewind-pfe${PLATFORM}:${TAG}
```

Comment:
Since we use these image names several times in this file could we define constants for the names?
https://github.com/eclipse/codewind-installer/pull/294/files#diff-02a4c9edab65df5dfdd065f5b995d1faR510
This could then be `${PFE_IMAGE_NAME}${PLATFORM}:${TAG}`
And the array of names could be:
```
baseImageNameArr = []string{
  PFE_IMAGE_NAME,
  PERFORMANCE_IMAGE_NAME,
}
```